me.marzeq.xbox-cloud-gaming-electron.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: me.marzeq.xbox-cloud-gaming-electron
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+separate-locales: false
+command: start-xbox-cloud-gaming-electron
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+
+modules:
+  - name: xbox-cloud-gaming-electron
+    buildsystem: simple
+    build-commands:
+      - unzip -d xbox-cloud-gaming-electron xbox-cloud-gaming-electron_*.zip
+      - rm xbox-cloud-gaming-electron_*.zip
+      - cp -r xbox-cloud-gaming-electron /app
+      - install -Dm 644 me.marzeq.xbox-cloud-gaming-electron.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm 644 me.marzeq.xbox-cloud-gaming-electron.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 644 me.marzeq.xbox-cloud-gaming-electron.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      - install -Dm 755 start-xbox-cloud-gaming-electron.sh /app/bin/start-xbox-cloud-gaming-electron
+    sources:
+      - type: file
```

Comment:
you can make this `archive` and you won't have to unzip. You can even set `dest`.

Reply:
I'm not sure if I've done this correctly, but this should now be fixed